docs/guides/upgrade-guide-agents-to-workers.md

```diff
@@ -0,0 +1,101 @@
+
+2. **Configuring remote flow code storage:** 
+    
+    storage blocks --> [pull action](/concepts/deployments/#the-pull-action)
+    
+    And you can still use an existing [storage block as your pull action](/guides/deployment/storage-guide/)!  The more general pull steps specification allows for explicit customization of the job that defines each run (for example, by adding an additional step that sets a working directory).
+
+3. **Configuring flow run infrastructure:** 
+    
+    run-infrastructure blocks --> [typed work pool](/concepts/work-pools/#worker-types) 
+    
+    Default infra config is now set on the typed work pool, and can be overwritten by both individual deployments or individual runs.
+
+4. **Managing multiple deployments:**
+    
+    Create and/or update many deployments at once through a [`prefect.yaml`](/concepts/deployments/#working-with-multiple-deployments) file.
+
+
+## What's the same
+
+- Storage blocks can be set as the pull action in a `prefect.yaml` file.
+- Infrastructure blocks have similar configuration fields as typed work pools.
+- Deployment-level infra-overrides operate in much the same way. 
+
+    `infra_override` -> [`job_variable`](/concepts/deployments/#work-pool-fields)
+
+- The process for starting an agent and [starting a worker](/concepts/work-pools/#starting-a-worker) in your environment are virtually identical.
+    
+    `prefect agent start --pool <work pool name>` --> `prefect worker start --pool <work pool name>`
+
+    !!! Tip "If you use infrastructure-as-code"
+        Notice the command to start a worker is very similar to the command to start an agent. If you previously used terraform, a [helm chart](https://github.com/PrefectHQ/prefect-helm/tree/main/charts/prefect-worker), or other infrastructure-as-code methods to start an agent, you should be able to continue using it for a worker, provided all uses of `agent` are changed to `worker`.
+
+## How to upgrade quickly
+
+If you have existing deployments that use infrastructure blocks, you can quickly upgrade them to be compatible with workers by following these steps:
+
+1. [Create a work pool](/concepts/work-pools/#work-pool-configuration) of the same type as the infrastructure block you are currently using.
+
+    1. Any work pool infrastructure type other than `Prefect Agent` will work.
+    2. Referencing the configuration you've set on the infrastructure block, set similar flow run infrastructure configuration on the work pool.
+
+2. [Start a worker](/concepts/work-pools/#starting-a-worker) to poll this work pool. You should see the command to start the worker as soon as you save your new work pool. 
+
+    ```
+    prefect worker start -p <work pool name>
+    ```
+
+3. [Deploy your flow](/#step-5-deploy-the-flow) guided by the deployment creation wizard:
+
+    !!! warning "Always run `prefect deploy` commands from the **root** level of your repo!"
```

Comment:
Could I get your eyes on the wording of this warning and note block?

@discdiver